backend/inspirehep/submissions/views.py

```diff
@@ -116,6 +117,29 @@ def populate_and_serialize_data_for_submission(
         return serialized_data
 
 
+class ConferenceSubmissionsResource(BaseSubmissionsResource):
+    decorators = [login_required_with_roles()]
+
+    def post(self):
+        """Adds new conference record"""
+        data = conference_loader_v1()
+        if "$schema" not in data:
```

Comment:
well this will never be set actually by the form, so maybe it's better to move logic to populate `$schema` to marshmallow loader?

Reply:
I removed it also for jobs